json_database/crypto.py

```
@@ -0,0 +1,112 @@
+        data = json.loads(data)
+    if len(key) > 16:
+        key = key[0:16]
+    ciphertext = unhexlify(data["ciphertext"])
+    if data.get("tag") is None:  # web crypto
+        ciphertext, tag = ciphertext[:-16], ciphertext[-16:]
+    else:
+        tag = unhexlify(data["tag"])
+    nonce = unhexlify(data["nonce"])
+    try:
+        return decrypt(key, ciphertext, tag, nonce)
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)